src/components/App/App.js

```diff
@@ -34,34 +34,46 @@ class App extends React.Component {
      return polish.brand === newPolish.brand
    })
 
-   const index = this.state.polishes.findIndex(foundPolish => {
-     return foundPolish.brand === newPolish.brand
-   })
-   let copy = [...this.state.polishes]
-
-   if (foundPolish) {
-     const foundColor = foundPolish.colors.find(color => {
-       return color.colour_name === newPolish.colorway
-     })
-     if (!newPolish.hue && foundColor) {
-      newPolish.hue = foundColor.hex_value
+   if (!foundPolish) {
```

Comment:
You could potentially DRY up this conditional and the find iteration above by making it a !== and a filter. Then you'll have your list of all polishes and can set in start without additional conditionals (I think lol)

Reply:
I'll have to play around with this! Great idea!